src/redux/books/bookSlice.js

```diff
@@ -1,7 +1,32 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  books: [],
+  books: [
```

Comment:
- [ ] You did a great job connecting the redux store with the components. :+1: My only concern is regarding the Author selector. I think that should be an input field instead of a selector since we don't have any idea which books the user is trying to add. Kindly change it to an input field. 🧐️

![image](https://user-images.githubusercontent.com/17243325/229517598-eeed166f-58ef-4662-b1fa-acfd0232526b.png)